### PR TITLE
ci: add size-limit integration

### PR DESCRIPTION
**Pull request by SuperOleg39**
Hello!

At first, thanks for the perfect library, and very impressive 7 version release!

About PR.
We use `lru-cache` library in tens SSR applications, and worry about sizes of any libraries, which has included in client code.
I think, that `size-limit` can help early detect significant library weight regression, and easily measure impact of any code optimizations (e.g. removing backward compatibility code).

What do you think?

### CODE REVIEW

File: .github/workflows/size-limit.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: "size"
+on:
+  pull_request:
+    branches:
+      - master
```
Comment (isaacs) [inline .github/workflows/size-limit.yml:5]: s/master/main/

**Comment by isaacs** [comment]
Failing with `Error: Parameter token or opts.auth is required`

Do you happen to know what's missing?  I'd love to catch file size increases, but I'm not familiar with this action.

File: .github/workflows/size-limit.yml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+jobs:
+  size:
+    runs-on: ubuntu-latest
+    env:
+      CI_JOB_NUMBER: 1
+    steps:
+      - uses: actions/checkout@v1
+      - uses: andresz1/size-limit-action@v1
+        with:
+          github_token: ${{ secrets.TINKOFF_BOT_PAT }}
```
Comment (isaacs) [inline .github/workflows/size-limit.yml:15]:
```suggestion
          github_token: ${{ secrets.GITHUB_TOKEN }}
```

**Comment by SuperOleg39** [comment]
Thanks for the fixes!

About `Error: Cannot find module '@size-limit/preset-small-lib'` error - this library creates comparsion with main branch, so fail in this PR is expected.

**Comment by isaacs** [comment]
I see.  So all it should need is the update to use the proper token, then?

**Comment by SuperOleg39** [comment]
> I see. So all it should need is the update to use the proper token, then?

I see a unexpected behaviour with `skip_step: build` in [action source code](https://github.com/andresz1/size-limit-action/blob/master/src/Term.ts#L34) - looks like this config will disable `install` command too.
Maybe will be better to remove `skip_step` option, also you already add empty "build" script.

After that, and after PR merge, i expect that next PR's will have a succesfull size-limit actions.

**Comment by isaacs** [comment]
> Maybe will be better to remove skip_step option, also you already add empty "build" script.

Another idea, since people will likely only care about this in client-side applications, is put a `webpack` build script in there, just for the purpose of building to test size.

Like, you probably don't care about whitespace, but if someone puts `require("path")` in there and blows up the bundle, that's a big deal, even though it's a tiny thing in the `index.js`.

**Comment by SuperOleg39** [comment]
Very good point!

**Pull request merged**